views/post_a_peep.erb

```diff
@@ -0,0 +1,8 @@
+Post a Peep
+<form action="/name" method="post">
+
+    <label for="name">Your peep:</label><br>
+    <input type="text" id="peep" name="peep"><br>
+    <input type="submit" value="Submit">
+
+</form>
```

Comment:
I really like that you thought of using a form. It looks clear in the browser that a user can 'Submit' a message. I didn't think of doing this!